lib/mainloop-control.c

```diff
@@ -183,8 +191,19 @@ control_connection_attach(ControlConnection *cc, GString *command, gpointer user
   log_stderr = new_values.log_stderr;
   msg_set_log_level(new_values.log_level);
 
+  if (start_debugger && !debugger_is_running())
+    {
+      //cfg_load_module(self->current_configuration, "mod-python");
+      debugger_start(main_loop, main_loop_get_current_config(main_loop));
```

Comment:
Note: checked the races around this, and I think everything already is taken care of by `console_acquire_from_fds()`.